client/src/index.jsx

```diff
@@ -6,7 +6,7 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      mainImages: [],
+      mainImages: ['https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fweknowyourdreams.com%2Fimages%2Fgoose%2Fgoose-08.jpg&f=1&nofb=1','https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.lQb88AzJ0ybvX9_AITaucAHaHa%26pid%3DApi&f=1','https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.lQb88AzJ0ybvX9_AITaucAHaHa%26pid%3DApi&f=1'],
```

Comment:
lol does this default to three rubber ducks in the slider????

Reply:
I'll let you wait and see when its deployed ;)